guides/tutorials/codelabs/pubsub-to-cf/index.md

```diff
@@ -62,10 +60,10 @@ b. Create a key for the service account and download the key in JSON format to y
 
 ## 4. Tell Spinnaker to Use the Pub/Sub Subscription
 
-a. Enable artifact support, then enable the GCS artifact provider:
+a. Enable the `artifactsRewrite` feature flag, then enable the GCS artifact provider:
```

Comment:
@fifthposition Does flipping this flag affect any of the rest of the codelab?

Reply:
Oh dear... looks like a change snuck in that I hadn't intended to commit. I'll revert this one. Sorry!